### ISSUE TITLE
Modify the icon badge in patient card 

### ISSUE THREAD

**Issue by nihal467** [coronasafe/care_fe#3261]
**Issue got updated on 20/11/2022**

![image](https://user-images.githubusercontent.com/57055998/202885658-c0133142-af03-482b-960d-ab755e84f3cf.png)

- inside the rectangle on the left side of the name of the patient, if patients have consultation filed and no bed assigned, show their "decision after consultation input"

![image](https://user-images.githubusercontent.com/57055998/202885826-e045332b-bb96-4678-8d9f-5ac488f527c2.png)

- The content in the box can be "Home isolation", "Admission", "Referred", "OP" and "Domiciliary Care"- the text may appear in smaller suttle text as compared to the bed numbers in Bold.

- For patients that are only registered and no consultation is filed, let's have No Consultation filed, and let's also make the box red. With this, we can get rid of the label that we have otherwise on the right side for no consultation filed.

**Comment by aparnacoronasafe** [comment]
Let's revert the design to the old one. 

Let there be evenly sized rectangles on the left side of the name of patients for all .
- Where bed is allocated, we show location and bed no. in the box
- For patients that have consultation filed and no bed assigned, show their "decision after consultation input"
<img width="757" alt="Screenshot 2022-08-03 at 6 00 34 PM" src="https://user-images.githubusercontent.com/67050716/182608770-d260eb12-d86f-4688-99b0-bb6262ffcc2c.png">
The content in the box can be "Home isolation", "Admission", "Referred", "OP" and "Domiciliary Care"- the text may appear in smaller suttle text as compared to the bed numbers in Bold.

- For patients that are only registered and no consultation is filed, lets have No Consultation filed and lets also make the box in red. With this we can get rid of the lable that we have otherwise on the right side for no consultation filed. 

This is my suggestion.

@gigincg @bodhish @nihal467 What do you guys think?

**Comment by nihal467** [comment]
@aparnacoronasafe cool it works

**Comment by nihal467** [comment]
@developedBySJ don't close this issue once PR #3324 is merged, as we have one more issue mentioned above, that needs to build regarding the content for patients when beds are not assigned, which can be another PR, in which you or others can work!

**Comment by aparnacoronasafe** [comment]
@developedBySJ please do work on the above issue, or unassign yourself so that someone else can work.

**Comment by aparnacoronasafe** [comment]
@khavinshankar DONOT close the issue. Even though patient page is redisigned, The bed no. badges are not added .

Please be very careful to read through the issue to see if all the tasks are completed. 

The bed badges with the above specifications needs to be added to the new patient UI.

**Comment by nihal467** [comment]
@developedBySJ the issue is not relevant anymore
